pkgs/by-name/gk/gk-cli/package.nix

```
@@ -0,0 +1,50 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+  coreutils,
+  installShellFiles,
+  makeWrapper,
+}:
+
+let
+  version = "2.1.0";
+in
+stdenv.mkDerivation {
+  pname = "gk-cli";
+  inherit version;
+
+  src = fetchzip {
+    url = "https://github.com/gitkraken/gk-cli/releases/download/v${version}/gk_${version}_Linux_x86_64.zip";
+    hash = "sha256-BZO8MVFxIXXB+9v4yB9XcZ0E68xOIUL7Mot7PZ+5M1U=";
+    stripRoot = false;
+  };
+
+  nativeBuildInputs = [
+    installShellFiles
+    makeWrapper
+  ];
+
+  installPhase = ''
+    mkdir --parents $out/bin/
+    install ./gk $out/bin/
+
+    installShellCompletion --bash ./**/gk.bash
+    installShellCompletion --fish ./**/gk.fish
+    installShellCompletion --zsh ./**/_gk
+  '';
+
+  postFixup = ''
+    wrapProgram $out/bin/gk \
+      --prefix PATH : "${lib.makeBinPath [ coreutils ]}"
+  '';
+
+  meta = {
+    description = "CLI for Git collaboration across multiple repos and services";
+    homepage = "https://www.gitkraken.com";
+    license = lib.licenses.unfree;
+    maintainers = [ lib.maintainers.pinage404 ];
+    mainProgram = "gk";
+    platforms = [ "x86_64-linux" ];
+  };
+}
```

Remember to close the newly opened parenthesis on the `mkDerivation` line:
```suggestion
})
```